services/order.py

```diff
@@ -0,0 +1,35 @@
+from django.db import transaction
+from django.db.models import QuerySet
+
+from db.models import Order, Ticket, User, MovieSession
+
+
+def create_order(tickets: list, username: str, date: str = None) -> Order:

```

Comment:
Using mutable default arguments like `list` can lead to unexpected behavior. Consider using `None` as the default and initializing the list inside the function.